common/src/main/java/com/wynntils/core/consumers/overlays/annotations/OverlayInfo.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2022-2023.
+ * Copyright © Wynntils 2022-2024.
```

Comment:
Revert this change, since the file is not edited.

Reply:
Because of the revert commit, this will be reverted.

Reply:
Let's see what happens! :)

Reply:
Seems like the undoing stuck.